atuin/src/command/client/history.rs

```diff
@@ -475,3 +553,52 @@ impl Cmd {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use eyre::Result;
+    use time::UtcOffset;
+
+    use super::Timezone;
+
+    #[test]
+    fn can_parse_local_timezone_spec() -> Result<()> {
+        let local_tz = Timezone(UtcOffset::current_local_offset()?);
+
+        assert_eq!(Timezone::from_str("l")?, local_tz);
+        assert_eq!(Timezone::from_str("local")?, local_tz);
+        Ok(())
+    }
```

Comment:
Also, this test is failing on Linux. I looked at it with a debugger, and noticed a contradiction between `time`'s code and documentation. I'll submit an issue there to ask about it.

Reply:
I did some digging, and it appears that there's no mistake in `time`; rather it's my own misunderstanding. Still though, I have to find a way to fix this test. Has to do with some really annoying thread-safety issues so it might take me a while to get it right.

Reply:
So I am going down this [***massive*** rabbit hole](https://github.com/time-rs/time/issues/293) that basically boils down to "there's no thread-safe way to get a system's timezone on POSIX systems". To be honest I'm completely flustered at this point.

To deal with this, `time` has implemented [some rather stringent checks](https://github.com/time-rs/time/blob/v0.3.31/time/src/sys/local_offset_at/unix.rs#L129-L154) to ensure safety. This means this test either has to run in a single-threaded environment, or we have to accept unsoundness (which admittedly can only ever happen if libc's `setenv` function is called directly without locking, but still). I'll see what I can do tomorrow.